tools/nix/parts/tooling.nix

```diff
@@ -11,6 +11,7 @@ _: {
         config.packages.bats
         commitlint
         curl
+        shfmt
```

Comment:
Sorted please, but if we go the `writeShellApplication` route we can omit it here.